src/utils/buildSchema.ts

```diff
@@ -0,0 +1,19 @@
+import { SchemaWrapper, LocalBusiness, FAQPage } from "@yext/pages-components";
+
+export function buildSchema(document: Record<string, any>) {
+  const localBusiness = document.address && {
+    ...LocalBusiness(document),
+    paymentAccepted: document.paymentOptions,
+    makesOffer: document.services,
+  };
+
+  const faqs =
+    document.c_faqSection?.linkedFAQs &&
+    FAQPage(document.c_faqSection.linkedFAQs);
+
+  const json = {
+    "@graph": [localBusiness, faqs].filter(Boolean),
```

Comment:
What's the filter doing?

Reply:
Removing falsy values so if faqs is null, it doesn't return [localBusiness, null]